cli/src/main/java/org/jboss/sbomer/cli/feature/sbom/command/DefaultProcessCommand.java

```diff
@@ -126,6 +141,54 @@ private void processBuild(Component component, Build build) {
         SbomUtils.addPedigreeCommit(component, build.getScmUrl() + "#" + build.getScmTag(), build.getScmRevision());
     }
 
+    private void processBrewBuild(Component component, Artifact artifact) {
+        log.debug("Looking up artifact '{}' in Brew...", artifact.getPublicUrl());
```

Comment:
I would like to have a log statement when the artifact in Brew:

a) was found
b) wasn't found

This will help debug "close" this log message with an outcome.